### DISCUSSION TITLE
v1.17.2 80 port issue

### DISCUSSION THREAD

**Discussion by DarkteK** [ddev/ddev#3024] [Q&A] [answered]
After I updated ddev to the newest version (v1.17.2) I'm encountering this error which I didn't face before:
```
ERROR: for ddev-router  Cannot start service ddev-router: Ports are not available: listen tcp 127.0.0.1:80: bind: An attempt was made to access a socket in a way forbidden by its access permissions.

ERROR: for ddev-router  Cannot start service ddev-router: Ports are not available: listen tcp 127.0.0.1:80: bind: An attempt was made to access a socket in a way forbidden by its access permissions.
Encountered errors while bringing up the project.
Failed to start ustma-members-d9: failed to start ddev-router: Failed to run docker-compose [-f /home/darktek/.ddev/.router-compose-full.yaml -p ddev-router up -d], err='exit status 1', stdout='', stderr='Starting ddev-router ...
Starting ddev-router ... error

ERROR: for ddev-router  Cannot start service ddev-router: Ports are not available: listen tcp 127.0.0.1:80: bind: An attempt was made to access a socket in a way forbidden by its access permissions.

ERROR: for ddev-router  Cannot start service ddev-router: Ports are not available: listen tcp 127.0.0.1:80: bind: An attempt was made to access a socket in a way forbidden by its access permissions.
Encountered errors while bringing up the project.'
```

I used `netstat -o -n -a | findstr 127.0.0.1:80` but nothing came up so I don't know how to continue from here, can someone help me please?

**Update 1:** I updated the **router_http_port** to be "**8080**" in **.ddev/config.yaml** but I'm still facing the same issue:
`router_http_port: "8080"`

**Update 2:** I had to restart my PC, besides changing the post to one specific profile. And it is working now, haven't tested the other projects without changing the ports, but at least it is working.

**Comment by rfay** [comment, marked as answer]
Please `ddev poweroff` and then just start this project. I imagine you have more than one project. Same question seemed to be asked in https://stackoverflow.com/questions/67736473/container-complains-of-port-443-after-changing-to-8443

**Reply by DarkteK** [reply]
Thanks!!

**Reply by rfay** [reply]
@DarkteK also note https://github.com/drud/ddev/issues/2981 - there's a bug in current docker 3.3.3.

**Reply by DarkteK** [reply]
Thanks @rfay !
